tests/integration_test/__tests__/sign-request.test.js

```diff
@@ -86,3 +86,75 @@ test('test sign request', async() => {
 		await file.dispose();
 	}
 });
+
+test('test sign request webhook', async() => {
```

Comment:
This should not be here. You do not need SignRequest to create and remove webhook. Maybe just create it-tests for webhooks?